operations_on_datastructures/reverse_a_queue_using_a_stack.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file 
+ * @author Alec Rothenberg 
+ * @brief Reverse a queue using a stack
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂